app/src/main/res/layout/view_contributor_cell.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        >
+
+    <data>
+
+        <variable
+                name="viewModel"
+                type="io.github.droidkaigi.confsched2017.viewmodel.ContributorViewModel"
+                />
+    </data>
+
+    <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?attr/selectableItemBackground"
+            android:clickable="true"
+            android:onClick="@{viewModel::onClickContributor}"
+            >
+
+        <ImageView
+                android:id="@+id/avatar"
+                android:layout_width="@dimen/contributor_image"
+                android:layout_height="@dimen/contributor_image"
+                android:layout_centerHorizontal="true"
+                android:layout_margin="@dimen/contributor_cell_margin"
```

Comment:
@konifar The image below is ImageView's margin top, left, right is 16dp and bottom is 8dp.
TextView's margin bottom is also 16dp. Your wish is my command, sir ;)

![contributors_design_fix02](https://cloud.githubusercontent.com/assets/1192694/22674976/34c87cd0-ed26-11e6-99b6-cb640d3c39b8.png)